Backend/OmegaSpring/src/main/java/com/project/omega/controller/AdminRoleController.java

```diff
@@ -44,7 +44,7 @@ public ResponseEntity getRoleByName(@PathVariable(value = "name") String name) {
     @DeleteMapping(value = "/{id}")
     public ResponseEntity deleteRoleById(@PathVariable(value = "id") Long id)  {
         AdminRoles iRole = roleService.deleteRoleById(id);
-        return new ResponseEntity(iRole, HttpStatus.I_AM_A_TEAPOT);
+        return new ResponseEntity(iRole, HttpStatus.GONE);
```

Comment:
`GONE` signifies an error, that `/role/{id}` was not found and it is likely permanent. Please return `OK` if successful

Check this: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE